http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java

```diff
@@ -262,7 +271,9 @@ public void testBadPort()
                 .setUri(new URI(scheme, null, host, 70_000, "/", null, null))
                 .build();
 
-        executeExceptionRequest(config, request);
+        assertThatThrownBy(() -> executeExceptionRequest(config, request))
+                .isInstanceOf(RuntimeException.class)
+                .hasMessageMatching(".*port out of range.*");
```

Comment:
is .* needed?
If yes - I saw somewhere in this PR that the message lacks them.

Reply:
This is a direct rewrite so we can improve that as a follow-up